deployment/Install-Client.ps1

```diff
@@ -162,6 +162,14 @@ function Install-Client {
         Write-ClientDebugLog -Message 'Uploading build content to the `$web storage container'
         # Turning Error only mode as this cmd shows a warning which causes misconception that user needs to sign in to azure cli
         $result = az storage blob upload-batch -s 'build' -d '$web' --account-name $StaticWebsiteStorageAccount --only-show-errors | ConvertFrom-Json
+        
+        try{
+            az storage logging update --log rwd --retention 30 --services b --account-name $StaticWebsiteStorageAccount --only-show-errors
+        } 
+        catch {
+            Write-Output "Could not enable logging for static website storage account"
```

Comment:
Since we've handled this as a non-fatal error (and it is indeed non-fatal to deployment), few things coming to my mind: -
* Should we log it as Warning instead of a normal `Write-Output`?
* Any reason for omitting the `Write-ClientDebugLog` to log the complete exception?
* Could we consider writing a small doc (FAQ maybe) around how to do this manually and point the aka.ms url of that document here 🤔?
* Maybe rephrase the Warning to be more like "_Failed enable auditing for static website. This could be a security risk, please enable auditing for static website manually via `aka.ms/learn-lti-audit-client`_"

Reply:
I have created a link as per your suggestion. I will update the content shortly to include the manual steps for enabling auditing. I will update the error message shortly!

Reply:
I have added the link to the manual steps mentioned in the docs.microsoft guide for now. I hope that works. Can make it a part of our documentation as a follow-up.